src/github.com/getlantern/flashlight/logging/logging_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/getlantern/go-loggly"
 	"github.com/getlantern/golog"
-	"github.com/stretchr/testify/assert"
+	"github.com/getlantern/testify/assert"
```

Comment:
Yeah seems like there's something odd happening. I'm also somehow not able to even isolate pluggable_transports.go in this diff @oxtoacart, although I think that's my own fault with the too many files issue...